hack/util.sh

```diff
@@ -71,21 +71,14 @@ sti::log::error_exit() {
   exit "${code}"
 }
 
-os::util::sed() {
+s2i::util::sed() {
```

Comment:
This makes a sync between openshift and s2i more difficult, right? Do we really need this rename?